www/js/app.js

```diff
@@ -811,6 +811,9 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
     // Since late 2014, all ZIM files should use relative URLs
     var regexpImageUrl = /^(?:\.\.\/|\/)+(I\/.*)$/;
     var regexpMetadataUrl = /^(?:\.\.\/|\/)+(-\/.*)$/;
+    // Regex below finds images, scripts and stylesheets with ZIM-type metadata and image namespaces [kiwix-js #378]
+    // DEV: If you want to support more namsespaces, add them to the END of the character set [-I] (not to the beginning) 
```

Comment:
Little typo on "namsespaces" : "namespaces"
As the regexp is not trivial, I think it's necessary to describe it a bit more in the comment.

Reply:
OK, will see if I can come up with a clearer comment.